service.watchedlist/lib/watchedlist/watchedlist.py

```diff
@@ -568,6 +578,7 @@ def get_watched_xbmc(self, silent):
                           "id": 1})
                 if json_response.has_key('result') and json_response['result'] != None and json_response['result'].has_key('tvshows'):
```

Comment:
.has_key() method is deprecated and should not be used. Correct syntax is: `if key in dict`